pkg/containers/containers.go

```diff
@@ -39,12 +46,13 @@ func GetDigest(ctx context.Context, refstr string) (string, error) {
 // GetDigestFromRef returns the digest of a container image reference
 // from a name.Reference.
 func GetDigestFromRef(ctx context.Context, ref name.Reference) (string, error) {
-	desc, err := remote.Get(ref,
-		remote.WithContext(ctx),
-		remote.WithUserAgent(constants.UserAgent))
+	digest, err := crane.Digest(ref.String(),
```

Comment:
Why did we switch from using `remote` to using the `crane` CLI module?  I'm also wondering why we removed `WithUserAgent`...

Reply:
I'll add that `WithUserAgent` in the next commit. 
From `remote` we are getting a descriptor and that always returns digest of the image index irrespective of the platform.